Components/Navbar/navbar.jsx

```diff
@@ -14,12 +14,10 @@ export default function Navbar() {
               <i className="fa fa-caret-down"></i>
             </button>
             <div className="dropdown-content">
-            <a>Football</a>
-            <a>Cooking</a>
-            <a>Coding</a>
-              {/* <Link to="/topics/football">Football</Link>
-              <Link to="/topics/cooking">Cooking</Link>
-              <Link to="/topics/coding">Coding</Link> */}
+             <Link to="/articles?topic=coding">Coding</Link>
+             <Link to="/articles?topic=cooking">Cooking</Link>
+             <Link to="/articles?topic=football">Football</Link>
```

Comment:
By hard coding the topics here you'd have to add in any future categories manually from the code. But you want this to be dynamic so if the database  expands then the links will expand automatically. Other ways to do this would be to grab all the topics from the db and then loop through those topics to render a bunch of Links automatically.